meadow/client/nixpkgs-src.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Copy-paste from playground. Tbh maybe move the one from playground to the root and name it "old-purescript-nixpkgs.json` or something?